src/server/sample_data/routes/lib/load_data.js

```diff
@@ -0,0 +1,78 @@
+import readline from 'readline';
```

Comment:
Have you checked out the code that inserts data for the es_archiver?  wonder if its similar enough if it could be refactored to common shared code.

Reply:
Not sure if shared code is worth the effort. Just because they duplicate functionality today does not mean that will always be the case. Also, es_archiver functions differently, it is set up to read an entire directory. 

@alexfrancoeur had wanted the ability to also handle data sources that are streamed over http. So, `loadData`, will get more features in future enhancements.